<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPipeChildClass</class>
 <widget class="QMainWindow" name="TestPipeChildClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestPipeChild</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="m_sendBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_receiveBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>接收</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="m_sendText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="m_receiveText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testpipechild.qrc"/>
 </resources>
 <connections/>
</ui>
